proto/api.proto

```diff
@@ -52,38 +52,175 @@ service Api {
+      description: "Returns a partially signed txn signature to settle pnl on Drift";
+      summary: "settle pnl on Drift";
+      tags: ["Drift V2", "Perp", "Trade"];
+    };
+  }
+
+  rpc PostDriftSettlePNLs(PostDriftSettlePNLsRequest) returns (PostDriftSettlePNLsResponse) {
+    option (google.api.http) = {
+      post: "/api/v2/drift/perp/settle-pnls"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns a partially signed txn signature to settle pnls on Drift";
+      summary: "settle pnls on Drift";
+      tags: ["Drift V2", "Perp", "Trade"];
+    };
+  }
+
+  rpc GetDriftAssets(GetDriftAssetsRequest) returns (GetDriftAssetsResponse) {
+    option (google.api.http) = {
+      get: "/api/v2/drift/assets"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns details of assets a user has on Drift";
+      summary: "details of assets a user on Drift";
+      tags: ["Drift V2", "Perp", "Trade"];
+    };
+  }
+
+  rpc GetDriftPerpContracts(GetDriftPerpContractsRequest) returns (GetDriftPerpContractsResponse) {
+    option (google.api.http) = {
+      get: "/api/v2/drift/perp/contracts"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns details of available perp contracts on Drift";
+      summary: "details of available perp contracts on Drift";
+      tags: ["Drift V2", "Perp", "Market"];
+    };
+  }
+
+  rpc PostLiquidateDriftPerp(PostLiquidateDriftPerpRequest) returns (PostLiquidateDriftPerpResponse) {
+    option (google.api.http) = {
+      post: "/api/v2/drift/perp/liquidate"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns a partially signed txn signature to liquidate perp positions on Drift";
+      summary: "liquidate perp positions";
+      tags: ["Drift V2", "Perp", "Trade"];
+    };
+  }
+
+  rpc GetDriftOpenPerpOrder(GetDriftOpenPerpOrderRequest)  returns (GetDriftOpenPerpOrderResponse) {
+    option (google.api.http) = {
+      get: "/api/v2/drift/perp/open-order"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns the detail of an open perp order either by ID or ClientOrderID";
+      summary: "PERP order";
+      tags: ["Drift V2", "Trade", "Perp", "Orderbook"];
+    };
+  }
+
+  rpc GetDriftOpenMarginOrder(GetDriftOpenMarginOrderRequest)  returns (GetDriftOpenMarginOrderResponse) {
+    option (google.api.http) = {
+      get: "/api/v2/drift/margin/open-order"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns the detail of an open margin order either by ID or ClientOrderID";
+      summary: "Margin open order";
+      tags: ["Drift V2", "Trade", "Margin"];
```

Comment:
Margin -> Margin V2. I understand we didn't really have Margin before so it will not get in the same group as V1 endpoints but I just think we can keep the approach for consistency and when we remove support of V1 we can just Find and replace all V2 in the tags here

Reply:
this is cleaned up now.